src/radical/pilot/raptor/__init__.py

```diff
@@ -3,4 +3,3 @@
 from .worker         import Worker
 from .worker_mpi     import MPIWorker
 from .worker_default import DefaultWorker
-
```

Comment:
Our coding guidelines actually want an empty line at end of file...

Reply:
I used our `.pylint` and `.flake8` files to check and they did not complain.

Reply:
fixed.